src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -259,6 +260,8 @@ class ArrayAttestationTrustStore : public AttestationTrustStore
     const size_t mNumCerts;
 };
 
+class DeviceAttestationRevocationDelegate;
```

Comment:
This would not be needed if DeviceAttestationRevocationDelegate was a subclass of DeviceAttestationVerifier

Reply:
Logged an issue to make it a subclass, will do it in follow-up https://github.com/project-chip/connectedhomeip/issues/34495